.travis.yml

```diff
@@ -109,6 +109,36 @@ jobs:
        - cd ../..
      after_success:
        - make style-py && git diff --exit-code;
+# Julia
+   - name: "Julia"
+     language: julia
+     os: linux
+     julia: 1
+     install:
+       - make info && make -j4
+       - export JULIA_LIBCEED_LIB=$(find $PWD/lib -name "libceed.*")
+       - pushd julia/LibCEED.jl
+     script:
+       - julia --project -e 'import Pkg; Pkg.build(); Pkg.test("LibCEED"; coverage=true)'
+       - unset JULIA_LIBCEED_LIB && julia --project -e 'import Pkg; Pkg.build(); Pkg.test("LibCEED")'
+       - >
+         julia --project=.style/ -e 'import Pkg; Pkg.instantiate()'
+         && julia --project=.style/ .style/ceed_style.jl
+         && git diff --exit-code src test examples
+     after_success:
+       - popd
+       - julia -e 'import Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder("julia/LibCEED.jl/src"))'
+   - stage: "Documentation"
```

Comment:
Two questions

a) Can we name this documentation job something like "Julia Documentation"

b) Why Julia 1 instead of 1.5? 1.5 seems to run faster on my other project and it seems reasonable to test with the latest released Julia.

Reply:
Also, can we call the stage "documentation" with a lower case d? The default testing stage is named "test" with a lower case t.

Reply:
>a) Can we name this documentation job something like "Julia Documentation"

Yes, sure, I'll change it.

> b) Why Julia 1 instead of 1.5? 1.5 seems to run faster on my other project and it seems reasonable to test with the latest released Julia.

1 means the latest stable 1.x.y release, so I believe it is running 1.5.2 automatically.